### PR TITLE
Develop database code

### PR DESCRIPTION
**Pull request by sunju0123**
**DatabaseHelper**

- `getUrlListFromQuery` : update query to get all database item.
- add `setExpectedTime` method.
- change `insertScriptedData` query column name.
- remove wordCount
- update insert query

**scriptedURL**
-  remove calling crawler code.

### CODE REVIEW

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java [hunk 1]
```diff
@@ -170,26 +178,26 @@ public void getTableAsString() {
 
   /**
    * Please Check URL duplication before using insert method.
-   * Using Duplication()
+   * Using isDuplicatedUrl()
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: Please fix the JavaDoc.

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java [hunk 2]
```diff
@@ -170,26 +178,26 @@ public void getTableAsString() {
...
     String escapedContent = StringEscapeUtils.escapeHtml4(content);
-    dataBase.execSQL("INSERT INTO PAGE (URL, TITLE, CONTENT, EXPECTEDTIME, IMGURL) VALUES ('" + url + "',\"" + escapedTitle + "\", \"" + escapedContent + "\", " + String.valueOf((int)expectedTime) + " , '" + imgUrl + "');");
+    dataBase.execSQL("INSERT INTO PAGE (URL, TITLE, CONTENT, EXPECTEDTIME, repImage) VALUES ('" + url + "',\"" + escapedTitle + "\", \"" + escapedContent + "\", " + String.valueOf((int)expectedTime) + " , '" + imgUrl + "');");
     dataBase.close();
   }
 
   /**
    * Please Check URL duplication before using insert method.
-   * Using Duplication()
+   * Using isDuplicatedUrl()
```
Comment (xhae) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: Please fix the JavaDoc.

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java (hunk 1 above)
Comment (sunju0123) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: done

File: Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java (hunk 2 above)
Comment (sunju0123) [inline Roys/app/src/main/java/com/dwg_karrier/roys/DataBaseOpenHelper.java]: done

**Comment by xhae** [comment]
Resolve conflicts please

**Pull request merged**